src/routes/votings/[id]/+page.svelte

```diff
@@ -1,9 +1,495 @@
 <script lang="ts">
+	import {
+		Breadcrumb,
+		BreadcrumbItem,
+		Button,
+		Modal,
+		Tag,
+		TextInput,
+		ToggleSkeleton
+	} from 'carbon-components-svelte';
+	import { ArrowRight, Add, Search } from 'carbon-icons-svelte';
+	import VotingResultTag from '$components/VotingResultTag/VotingResultTag.svelte';
+	import DownloadData from '$components/DownloadData/DownloadData.svelte';
+	import BillCategoryTag from '$components/BillCategoryTag/BillCategoryTag.svelte';
+	import BillCard from '$components/BillCard/BillCard.svelte';
+	import { DefaultVoteOption, type CustomVoteOption, DefaultVotingResult } from '$models/voting.js';
+	import { onMount } from 'svelte';
 	export let data;
-</script>
 
-<a href="{data.voting.id}/votes">Votes page</a>
+	let open = false;
+	let selectedTab = '';
+	let selectedMenu = 'summary';
```

Comment:
You use `summary`, `byParty`, `byPerson` many times in the code and it can easily cause an error because of the typo. I suggest you create a TypeScript `enum` that has values of `summary`, `byParty`, `byPerson`. Then you can type `selectedMenu` as an enum, and use an enum value instead of a string.